### PR TITLE
[WIP] Propose updates to the SIG ContribEx Charter

### PR DESCRIPTION
**Pull request by castrojo**
This is a work in progress and needs community discussion.

Things that end up not belonging to anyone end up being owned by Contrib Ex. This is a first draft at nailing down exactly what we should be responsible for and what we should not be responsible for.

Note that this effectively removes ContribEx as the default moderator of all the kubernetes properties. The project will still need this work to get done, so where appropriate I've moved that back to the SIGs, and for everything else it will require discussion or perhaps a new community structure to manage those things. 

Changes: 

- Explicitly add Kubernetes Gsuite as a responsibility
- Explicitly add CNCF as the lead owner of Zoom, YouTube, and Slack
accounts
- Begin to document responsibilities around contributor summits between
this SIG and the CNCF
- Explicitly remove moderation of social properties as a responsility
- Explicitly remove closing down working groups as a responsibility
- Explicitly remove SIG Calendar, lists, YouTube playlists, and Zoom
settings as a responsibility

/hold
/sig contributor-experience

### CODE REVIEW

File: sig-contributor-experience/charter.md [hunk 1]
```diff
@@ -76,14 +70,29 @@ CNCF in many of the noted cases above, contributes funding to our platforms, pro
...
+  - We do not close down Working Groups, that is the responsibility of the sponsoring SIGs of that working group
 - We are not the [code of conduct committee] and therefore do not control incident management reporting or decisions; however, our moderation guidelines allow us to act swiftly if there is a clear violation of terms of either our code of conduct or one of our supported platforms terms of service. If there is an action that the committee needs to take that involves one of these platforms (example: the removal of someone from GitHub), we will carry that out if none of the committee members have access.
+- SIG Properties 
+  - SIG Calendars and their settings
+  - SIG Mailing List permissions, moderation, and maintenance
+  - SIG YouTube playlist videos
+  - SIG settings on their Zoom account
+  - We are available to assist SIGs with these properties, but the responsibility of managing these day to day belongs to the individual SIG
+- Account ownership of the Kubernetes public social properties. The CNCF may delegate admin or management positions on these properties to members of Contrib Ex:
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: can we say contributor experience instead of contribex

File: sig-contributor-experience/charter.md [hunk 2]
```diff
@@ -76,14 +70,29 @@ CNCF in many of the noted cases above, contributes funding to our platforms, pro
...
+  - Slack or YouTube moderation (This should be a new SIG or another group)
 - The contributor experience for repos not included in the Kubernetes associated repositories list found in the [GitHub Management] subproject README.
-- Steering committee election policy updates and maintenance.
+- Steering committee election policy updates and maintenance. (Steering Comittee handles this)
 - We do not create SIGs/WGs but can assist in the various community management needs of their micro communities that would kick off their formation and keep them going.
+  - We do not close down Working Groups, that is the responsibility of the sponsoring SIGs of that working group
 - We are not the [code of conduct committee] and therefore do not control incident management reporting or decisions; however, our moderation guidelines allow us to act swiftly if there is a clear violation of terms of either our code of conduct or one of our supported platforms terms of service. If there is an action that the committee needs to take that involves one of these platforms (example: the removal of someone from GitHub), we will carry that out if none of the committee members have access.
+- SIG Properties
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: i feel like this is in sig governance docs but if it makes sense to call out here so be it

File: sig-contributor-experience/charter.md [hunk 3]
```diff
@@ -12,21 +12,15 @@ We do this by listening - whether it’s through our roadshows to SIG meetings,
...
+- [GitHub Management]
+- [Mailing lists] / Google groups for the project as a whole (eg: kubernetes-dev@googlegroups.com) that are not owned by SIGs
 - Work with other SIGs and interested parties in the project to execute GitHub tasks where required, see [GitHub Management] for more detail
 - Own and execute events that are targeted to the Kubernetes contributor community, including:
   - The weekly [Kubernetes Community meeting]
-  - [Contributor Summit(s)]
+  - Maintenance of the Kubernetes Gsuite
+    - Managing @kubernetes.io aliases for project usage
+    - Managing the Kubernetes Community Calendar
+  - Content and Format of the [Contributor Summit(s)]
```
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: design decisions

File: sig-contributor-experience/charter.md (hunk 1 above)
Comment (parispittman) [reply sig-contributor-experience/charter.md]: can this section be combined with the below? i think they are saying the same things

File: sig-contributor-experience/charter.md [hunk 4]
```diff
@@ -76,14 +70,29 @@ CNCF in many of the noted cases above, contributes funding to our platforms, pro
...
+- Account ownership of the Kubernetes public social properties. The CNCF may delegate admin or management positions on these properties to members of Contrib Ex:
+  - Main Zoom account (CNCF)
+  - Main YouTube account (CNCF)
+  - Main Slack ownership (CNCF)
 - Communication platforms that are out of our scope for maintenance and support but we may still have some influence:
     - [r/kubernetes]
     - [@kubernetesio] twitter account
     - [kubernetes blog]
+- Logistical Execution of the [Contributor Summit(s)]
+  - The CNCF will handle the logistical requirements of the contributor summits with our input while we will focus on the content.
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: the cncf will provide us with an events manager who will handle the event logistics and provide guidance with the requirements we set forth

File: sig-contributor-experience/charter.md [hunk 5]
```diff
@@ -12,21 +12,15 @@ We do this by listening - whether it’s through our roadshows to SIG meetings,
 
 #### Code, Binaries and Services
 
-- Establish policies, standards and procedures for the use, [moderation], and management of all public platforms officially used by the project, including but not limited to:
-  - [discuss.kubernetes.io]
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: this is out of our scope and should live in the home with slack

File: sig-contributor-experience/charter.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: and running the meetings and providing project management

File: sig-contributor-experience/charter.md [hunk 6]
```diff
@@ -76,14 +70,29 @@ CNCF in many of the noted cases above, contributes funding to our platforms, pro
 - Code for the testing and CI infrastructure - that’s SIG Testing
 - [kubernetes/community]  ownership of folders for KEPs and Design Proposals. Members are to follow those folders owners files and SIG leadership for the specific issue/PR in question.
 - User community management. We hold office hours because contributors are a large portion of the volunteers that run that program.
+  - Slack or YouTube moderation (This should be a new SIG or another group)
```
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: our youtube channel is primarily contributor use though and will be happy to have it stay with us - we also stream and would be a roadblock to have another owner here 

slack is not contributor majority and should indeed be owned by another body

File: sig-contributor-experience/charter.md [hunk 7]
```diff
@@ -76,14 +70,29 @@ CNCF in many of the noted cases above, contributes funding to our platforms, pro
...
 - The contributor experience for repos not included in the Kubernetes associated repositories list found in the [GitHub Management] subproject README.
-- Steering committee election policy updates and maintenance.
+- Steering committee election policy updates and maintenance. (Steering Comittee handles this)
 - We do not create SIGs/WGs but can assist in the various community management needs of their micro communities that would kick off their formation and keep them going.
+  - We do not close down Working Groups, that is the responsibility of the sponsoring SIGs of that working group
 - We are not the [code of conduct committee] and therefore do not control incident management reporting or decisions; however, our moderation guidelines allow us to act swiftly if there is a clear violation of terms of either our code of conduct or one of our supported platforms terms of service. If there is an action that the committee needs to take that involves one of these platforms (example: the removal of someone from GitHub), we will carry that out if none of the committee members have access.
+- SIG Properties 
+  - SIG Calendars and their settings
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: we are responsible for the upkeep of the shared calendar and making sure that is working

Comment (parispittman) [reply sig-contributor-experience/charter.md]: we also advise on settings and best practices in order to display on shared calendar or wherever

File: sig-contributor-experience/charter.md [hunk 8]
```diff
@@ -76,14 +70,29 @@ CNCF in many of the noted cases above, contributes funding to our platforms, pro
...
+  - SIG Mailing List permissions, moderation, and maintenance
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: we advise on this and come up with guidelines

File: sig-contributor-experience/charter.md [hunk 9]
```diff
@@ -76,14 +70,29 @@ CNCF in many of the noted cases above, contributes funding to our platforms, pro
...
+  - SIG YouTube playlist videos
+  - SIG settings on their Zoom account
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: we advise and set up guidelines

File: sig-contributor-experience/charter.md [hunk 10]
```diff
@@ -76,14 +70,29 @@ CNCF in many of the noted cases above, contributes funding to our platforms, pro
...
-- Steering committee election policy updates and maintenance.
+- Steering committee election policy updates and maintenance. (Steering Comittee handles this)
 - We do not create SIGs/WGs but can assist in the various community management needs of their micro communities that would kick off their formation and keep them going.
+  - We do not close down Working Groups, that is the responsibility of the sponsoring SIGs of that working group
 - We are not the [code of conduct committee] and therefore do not control incident management reporting or decisions; however, our moderation guidelines allow us to act swiftly if there is a clear violation of terms of either our code of conduct or one of our supported platforms terms of service. If there is an action that the committee needs to take that involves one of these platforms (example: the removal of someone from GitHub), we will carry that out if none of the committee members have access.
+- SIG Properties 
+  - SIG Calendars and their settings
+  - SIG Mailing List permissions, moderation, and maintenance
+  - SIG YouTube playlist videos
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: we advise and set up guidelines

File: sig-contributor-experience/charter.md [hunk 11]
```diff
@@ -12,21 +12,15 @@ We do this by listening - whether it’s through our roadshows to SIG meetings,
 
 #### Code, Binaries and Services
 
-- Establish policies, standards and procedures for the use, [moderation], and management of all public platforms officially used by the project, including but not limited to:
```
[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: these are the transparency tools of the project that steering works out in governance docs and should have operational responsibility with contributor experience. in the case where users are the majority, there needs to be some shared ownership, if not complete control and operational responsibility changed to them.

[the remark this answers is not in the document]
Comment (parispittman) [reply sig-contributor-experience/charter.md]: them = a body that has end user community at its mission

**Comment by castrojo** [comment]
@parispittman ok PTAL, I've restructured it so we explicitly split setting policy (which should be contribex) and then the list of all the properties, and then having the matching list in the out-of-scope section.

**Comment by castrojo** [comment]
I beleive this should cover most of your comments where ContribEx should have policy control of the properties.

File: sig-contributor-experience/charter.md [hunk 12]
```diff
@@ -1,51 +1,45 @@
...
-- Perform issue triage on and maintain the [kubernetes/community] repository.  
-- Help SIGs with being as transparent and open as possible through creating best practices, guidelines, and general administration of YouTube, Zoom, and our mailing lists where applicable
-- Assist SIGs/WG Chairs and Technical Leads with organizational management operations as laid out in the [sig-governance] doc
-- Distribute contributor related news on various Kubernetes channels, including Cloud Native Compute Foundation ([CNCF]) for posting blogs, social media, and other platforms as needed.
-- Establish and share metrics to measure project health, community health, and general trends, including:
-  - ongoing work with the CNCF to improve [DevStats]
-  - the contributor experience survey(s)
-  - engagement on project platforms that we manage
+  - Assist SIGs/WG Chairs and Technical Leads with organizational management operations as laid out in the [sig-governance] doc
+  - Distribute contributor related news on various Kubernetes channels, including Cloud Native Compute Foundation ([CNCF]) for posting blogs, social media, and other platforms as needed.
```
Comment (dankohn) [inline sig-contributor-experience/charter.md]: Cloud Native Computing Foundation (not Compute)

File: sig-contributor-experience/charter.md [hunk 13]
```diff
@@ -1,51 +1,45 @@
 # Contributor Experience Special Interest Group Charter
 
-This charter adheres to the conventions described in the [Kubernetes Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: why did you remove this? this is a part of the charter template.

File: sig-contributor-experience/charter.md [hunk 14]
```diff
@@ -71,26 +65,42 @@ We cross-cut all SIGs and WGs to deliver the following processes:
...
+    - [Kubernetes Blog]
+
 ### Out of scope
 
 - Code for the testing and CI infrastructure - that’s SIG Testing
-- [kubernetes/community]  ownership of folders for KEPs and Design Proposals. Members are to follow those folders owners files and SIG leadership for the specific issue/PR in question.
-- User community management. We hold office hours because contributors are a large portion of the volunteers that run that program.
-- The contributor experience for repos not included in the Kubernetes associated repositories list found in the [GitHub Management] subproject README.
-- Steering committee election policy updates and maintenance.
+- [kubernetes/community] Members are to follow those folders owners files and SIG leadership for the specific issue/PR in question.
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: what is those folders?

File: sig-contributor-experience/charter.md [hunk 15]
```diff
@@ -1,51 +1,45 @@
...
-  - Other events, like other SIG face to face events, upon request and consideration
-- Strategize, build, and execute on scalable [mentoring programs] for all contributor levels. These may include:
-  - [Google Summer of Code]
-  - [Outreachy]
-  - [Meet Our Contributors]
-  - [Group Mentoring - WIP]
-  - [The 1:1 Hour - WIP]
-  - Speed Mentoring sessions at selected KubeCon/CloundNativeCon's
-- Help onboard new and current contributors into the culture, workflow, and CI of our contributor experience through the [contributor guide], other related documentation, [contributor summits] and programs tailored to onboarding.
-- Perform issue triage on and maintain the [kubernetes/community] repository.
```
Comment (parispittman) [inline sig-contributor-experience/charter.md]: why remove this?

File: sig-contributor-experience/charter.md [hunk 16]
```diff
@@ -1,51 +1,45 @@
...
-  - [Meet Our Contributors]
-  - [Group Mentoring - WIP]
-  - [The 1:1 Hour - WIP]
-  - Speed Mentoring sessions at selected KubeCon/CloundNativeCon's
-- Help onboard new and current contributors into the culture, workflow, and CI of our contributor experience through the [contributor guide], other related documentation, [contributor summits] and programs tailored to onboarding.
-- Perform issue triage on and maintain the [kubernetes/community] repository.  
-- Help SIGs with being as transparent and open as possible through creating best practices, guidelines, and general administration of YouTube, Zoom, and our mailing lists where applicable
-- Assist SIGs/WG Chairs and Technical Leads with organizational management operations as laid out in the [sig-governance] doc
-- Distribute contributor related news on various Kubernetes channels, including Cloud Native Compute Foundation ([CNCF]) for posting blogs, social media, and other platforms as needed.
-- Establish and share metrics to measure project health, community health, and general trends, including:
```
[the remark this answers is not in the document]
Comment (castrojo) [reply sig-contributor-experience/charter.md]: For your last three comments: It shows as removed in the diff because I shuffled around the order, you have to switch to viewing the whole document to see the new order.

File: sig-contributor-experience/charter.md [hunk 17]
```diff
@@ -71,26 +65,42 @@ We cross-cut all SIGs and WGs to deliver the following processes:
...
+  - SIG Mailing List permissions, moderation, and maintenance
+  - SIG YouTube playlist videos
+  - SIG settings on their Zoom account
+- Ownership, staffing, and moderation of Kubernetes communication properties. The CNCF may delegate admin or management positions on these properties to members of Contributor Experience:
+  - [discuss.kubernetes.io] - This tool is mostly self-moderating and enough Kubernetes members participate that most of these roles are administrative.
+  - [Mailing lists] / Google groups (this belongs to the each SIG)
+  - [Slack] (CNCF)
+  - [/kubernetescommunity] YouTube channel (CNCF)
+  - [Zoom] (CNCF)
+  - Ownership of the Kubernetes Gsuite (this is Steering Committee)
```
Comment (idvoretskyi) [inline sig-contributor-experience/charter.md]: Doesn't it conflict with https://github.com/kubernetes/community/pull/3526/files#diff-ecc1c053d46fb60e012bf855c9630608R29?

Also, 'G Suite' (not 'Gsuite') :)

File: sig-contributor-experience/charter.md (hunk 14 above)
Comment (mrbobbytables) [reply sig-contributor-experience/charter.md]: Design proposals technically still live in k/community. They haven't been migrated to k/enhancements yet, but that is the intended home AFAIK.

File: sig-contributor-experience/charter.md [hunk 18]
```diff
@@ -1,51 +1,45 @@
...
 
-- Establish policies, standards and procedures for the use, [moderation], and management of all public platforms officially used by the project, including but not limited to:
+The inscope bullets listed below have a matching bullet for out of scope
+
+- Help onboard new and current contributors into the culture, workflow, and CI of our contributor experience through the [contributor guide], other related documentation, [contributor summits] and programs tailored to onboarding.
+- Strategize, build, and execute on scalable [mentoring programs] for all contributor levels. These may include:
+  - [Google Summer of Code]
+  - [Outreachy]
+  - [Meet Our Contributors]
+  - [Group Mentoring]
```
Comment (mrbobbytables) [inline sig-contributor-experience/charter.md]: group mentoring and the 1:1 hour links are broken. They're this at the bottom:
```
[Group Mentoring - WIP]:  https://git.k8s.io/community/mentoring/group-mentoring.md
[The 1:1 Hour - WIP]: https://git.k8s.io/community/mentoring/the1-on-1hour.md
```

**Comment by parispittman** [comment]
i brought up moving moderation to the code of conduct committee to get folks thoughts. we have a meeting next week. maybe a temp move until something else is found? unsure - wip thoughts.

File: sig-contributor-experience/charter.md (hunk 17 above)
Comment (castrojo) [reply sig-contributor-experience/charter.md]: One is ownership of the suite, which belongs to SC, the other is usage of the suite, which is contribex.

**Comment by castrojo** [comment]
Ok I've added fixes based on the comments I've resolved so far and made it more clear where we need a new SIG instead of just "CNCF".

**Comment by vllry** [comment]
👍

File: sig-contributor-experience/charter.md (hunk 13 above)
Comment (cblecker) [reply sig-contributor-experience/charter.md]: agree with @parispittman's comment

File: sig-contributor-experience/charter.md (hunk 14 above)
Comment (cblecker) [reply sig-contributor-experience/charter.md]: The intention of this is describing the fact that we own the repo as a whole, and the operational processes around the repo, but we delegate certain subfolders to other sigs/parts of the project. For example, we own the sig docs generator and the root, but we don't own the steering committee folder, or the the various sig folders. If we delegate it, we don't own it.

File: sig-contributor-experience/charter.md [hunk 19]
```diff
@@ -1,51 +1,45 @@
...
+  - [The 1:1 Hour]
+  - Speed Mentoring sessions at selected KubeCon/CloundNativeCon's
+- Establish policies, standards and procedures for the following:
   - [discuss.kubernetes.io]
-  - [GitHub Management]
+  - [GitHub Project Organizational Management]
```
Comment (cblecker) [inline sig-contributor-experience/charter.md]: What does this mean?

File: sig-contributor-experience/charter.md [hunk 20]
```diff
@@ -12,21 +12,15 @@ We do this by listening - whether it’s through our roadshows to SIG meetings,
...
-- Establish policies, standards and procedures for the use, [moderation], and management of all public platforms officially used by the project, including but not limited to:
-  - [discuss.kubernetes.io]
-  - [GitHub Management]
-  - [Mailing lists] / Google groups for the project as a whole (eg: kubernetes-dev@googlegroups.com) and for individual sigs and wgs where the Chairs have provided us ownership
-  - [Slack]
-  - [/kubernetescommunity] YouTube channel
-  - [Zoom]
-- Establish and staff teams responsible for the administration and moderation of these platforms
-  - Teams must be staffed by trusted contributors spanning time zones, see [moderation] for more detail
-  - They are authorized to take immediate action when dealing with code of conduct issues, see [moderators] for the full list
```
[the remark this answers is not in the document]
Comment (jberkus) [reply sig-contributor-experience/charter.md]: This seems like Jorge is proposing that Contribex not be responsible for ANY of the platforms.

**Comment by jberkus** [comment]
Jorge:

Can you spell out why you think we should remove each item, whether because:

1. You don't think that ContribEx should be responsible for them at all (e.g. WG shutdown)
2. We can't be responsible for them because of CNCF account ownership/controls, or
3. We can't staff them because not enough people

It might even be helpful if we break out the big question (moderating/owning community channels) from the other incidental responsibilities (WG, calendar, youtube).

Also, won't questions around mailing list ownership change if we get a community-controlled GSuite?

**Comment by castrojo** [comment]
/close

After IRL discussions at kubecon we're going to table this for now.
